intro-to-object-oriented-programming/what-is-oop.md

```diff
@@ -128,19 +129,29 @@ We might say...
 
 ### State and Behavior
 
-When we start defining our own classes, we will specify their **state** and their generic **behavior.**
+When we start defining our own classes, we will specify two main things: their **state** and their **behavior**.
+
+In the context of Object-Oriented Programming, **state** is used to refer to all the variables packaged within a class and **behavior** is used to refer to all the functions packaged within a class.
+
+### Attributes and Methods
+
+There are two more words we will often see in the context of OOP: **attribute** and **method**.
+
+For whatever reason, engineers long-ago decided we needed new words to describe variables and functions that make up a class's state and behavior. 
+
+An **attribute** is just a specific type of **variable**: one defined within a class. Similarly a **method** is just a specific type of **function**: one defined within a class.
 
-**State** describes any data or attributes that should be kept over its lifetime—from the moment we create an instance until we let the system reclaim its resources.
+These four words (state, behavior, attributes, and methods) aren't introducing anything new. Instead, they let us talk about things we already know (variables and functions) in a new context (being in a class)!
 
-**Behavior** describes any actions that objects of this class should be able to do.
+<img src="../assets/what-is-oop_class-breakdown.png" alt="Class Breakdown" width="800" height="500"/>
```

Comment:
Just so I'm making sure I'm learning correctly, are you using the `img` tag to get the specific width and height?

If so, I'll keep around `<img src="" alt="" min-width="800"/>` as a template for myself, probably?

Reply:
Yes, that's exactly why :)